azure-key-vault/microsoft_api_auth.py

```diff
@@ -0,0 +1,115 @@
+""" Copyright start
+  Copyright (C) 2008 - 2023 Fortinet Inc.
+  All rights reserved.
+  FORTINET CONFIDENTIAL & FORTINET PROPRIETARY SOURCE CODE
+  Copyright end """
+
+from requests import request
+from time import time, ctime
+from datetime import datetime
+from connectors.core.connector import get_logger, ConnectorError
+from connectors.core.utils import update_connnector_config
+from .const import *
```

Comment:
After changing the file name, please the "from .const import *" to "from .constants import *"